Mapping Tools/views/Sliderator/SlideratorView.xaml.cs

```diff
@@ -86,10 +86,9 @@ private void Import_Slider(object sender, RoutedEventArgs e)
                 Timing timing = beatmap.BeatmapTiming;
                 List<HitObject> markedObjects = selected;
 
-                try {
+                if(markedObjects.Find(s => s.IsSlider) != null)
```

Comment:
Now this is using Find twice. Using it once is better for performance and stuff.